.github/workflows/layout.html

```diff
@@ -0,0 +1,24 @@
+<!doctype html>
```

Comment:
Similarly, is there a reason this has been duplicated in this folder?

Reply:
Don't know how that happened, deleted now